src/app/features/dashboard-widgets/entity-count-dashboard-widget/entity-count-dashboard/entity-count-dashboard.component.ts

```diff
@@ -161,19 +162,22 @@ export class EntityCountDashboardComponent
   ): GroupCountRow[] {
     const groupByType = this._entity.schema.get(groupByField);
     const groups = groupBy(entities, groupByField as keyof Entity);
-    return groups.map(([group, entities]) => {
-      const label = extractHumanReadableLabel(group);
-      const groupedByEntity =
-        groupByType.dataType === EntityDatatype.dataType
-          ? groupByType.additional
-          : undefined;
-      return {
-        label: label,
-        value: entities.length,
-        id: group?.["id"] || label,
-        groupedByEntity: groupedByEntity,
-      };
-    });
+    return sortBy(
+      groups.map(([group, entities]) => {
+        const label = extractHumanReadableLabel(group);
+        const groupedByEntity =
+          groupByType.dataType === EntityDatatype.dataType
+            ? groupByType.additional
+            : undefined;
+        return {
+          label: label,
+          value: entities.length,
+          id: group?.["id"] || label,
+          groupedByEntity: groupedByEntity,
+        };
+      }),
+      (group) => group.id,
```

Comment:
My expectation for this, functionality wise, would be:
1. I edit the configurable-enum options (by editing a Child's details and using the tool icon of the "center" dropdown field), re-ordering the items in the dropdown edit dialog to make "Alipore" the last item in the dropdown list
2. I then go to the dashboard and see the items as disaggregations in that order, i.e. "Alipore" is listed last.

That's not the case yet.

I am not sure how the order of dropdown items is managed, but they support drag&drop when editing, so that component probably can give some context

Reply:
Ah, I understand. I tried to implement this, PTAL.